src/lib.rs

```diff
@@ -23,6 +25,7 @@ fn psqlpy(py: Python<'_>, pymod: &PyModule) -> PyResult<()> {
     pymod.add_class::<driver::common_options::ConnRecyclingMethod>()?;
     pymod.add_class::<query_result::PSQLDriverPyQueryResult>()?;
     add_module(py, pymod, "extra_types", extra_types_module)?;
+    add_module(py, pymod, "geometric_types", geometric_types_module)?;
```

Comment:
In my opinion, it's better to have all extra types in the `extra_types` module, otherwise user will search for them.

Reply:
I did this, because think that on implementing other postgres types - `extra types` module will be large and maybe hard to read. But i will add them to existing module.